app/assets/stylesheets/components/login_page.scss

```diff
@@ -0,0 +1,138 @@
+#devise-form {
+  
+  overflow: hidden;
+
+  h1 {
+    text-align: center;
+  }
+
+  .card {
+    width: 435px;
```

Comment:
Think about devices with width < 435px. Add `max-height` and `min-height` properties.